inst/rmarkdown/templates/ajs_outbreak/skeleton/skeleton.Rmd

```diff
@@ -1163,16 +1328,24 @@ mortality_rate(deaths, population, multiplier = 10000, mergeCI = TRUE) %>%
   kable(digits = 2)
 ```
 
+
+
+<!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+mortality_rate_CMR
+
 To produce a crude mortality rate attributable to AJS per 10,000 people per 
 day, use the folowing code chunk. 
+
 This assumes that you are capturing every AJS death in your population and 
 that your population remains stable over the time period of interest. 
+
 In this situation the time period of interest is from the beginning of the 
 epiweek your first case occured in, until the last day of the epiweek you are 
-currently reporting on. (see this [presentation](https://www.odi.org/sites/odi.org.uk/files/odi-assets/events-presentations/776.pdf) for more detail)
```

Comment:
remove hyperlink on purpose?

Reply:
hyperlink is in markdown, and this comment is not rendered in markdown